llvm/tools/llvm-profgen/PerfReader.cpp

```diff
@@ -53,6 +53,10 @@ static cl::alias
                           cl::desc("Comma-delimited version of -perf-event"),
                           cl::aliasopt(PerfEventFilter));
 
+static cl::opt<uint64_t>
+    SamplePeriod("sample-period", cl::init(1),
```

Comment:
This can be confusing -- having default value 1 implies we always assume a sample period of 1.